content/core-features/authentication.textile

```diff
@@ -21,15 +21,17 @@ redirect_from:
   - /general/_authentication_capabilities
 ---
 
-Both the "REST client library":/rest/authentication and the "Realtime client library":/realtime/authentication use common authentication mechanisms. The two schemes supported by Ably are "Basic Authentication":#basic-authentication, which uses your private "API key":https://knowledge.ably.com/what-is-an-app-api-key, and "Token Authentication":#token-authentication, which uses short-lived tokens for access which are periodically renewed.
+Both the "REST client library":/rest/authentication and the "Realtime client library":/realtime/authentication use common authentication mechanisms. The two schemes supported by Ably are "basic authentication":#basic-authentication, which uses your Ably "API key":https://knowledge.ably.com/what-is-an-app-api-key, and "token authentication":#token-authentication, which uses short-lived tokens for access. These tokens are periodically renewed.
 
-Token Authentication, in most cases, is the recommended strategy on the client-side as it provides more fine-grained access control and limits the risk of exposed or compromised credentials.
+**Important:** Token authentication is the recommended authentication mechanism on the client-side as it provides more fine-grained access control and limits the risk of exposing your Ably API key. In production systems you should never use basic authentication on the client-side as it exposes your Ably API key. You should never use token authentication server-side as this results in unnecessary overhead - the server would periodically require token requests to authenticate. For server-side, basic authentication is the recommended approach.
```

Comment:
I think it's worth stating explicitly that keys should be avoided client-side as they don't have an expiry, so a compromised key could be used indefinitely by an attacker, whereas there is only a small window of time during which a compromised token can be used.

Reply:
Good point. I will also add that to the Best Practice Guide.